File: VB/WorkflowDemoEF.Module/Objects/WorkflowDemoDbContext.vb
Imports System
Imports System.Collections.Generic
Imports System.Data.Common
Imports System.Data.Entity
Imports System.Linq
Imports System.Text
Imports DevExpress.ExpressApp.EF.Updating
Imports DevExpress.ExpressApp.Workflow.EF
Imports DevExpress.ExpressApp.Workflow.Versioning
Imports DevExpress.Workflow.EF

Namespace WorkflowDemoEF.Module.Objects
    Public Class WorkflowDemoDbContext
        Inherits DbContext

        Public Sub New()
            MyBase.New()
        End Sub
        Public Sub New(ByVal connectionString As String)
            MyBase.New(connectionString)
        End Sub

        Protected Overrides Sub OnModelCreating(ByVal modelBuilder As DbModelBuilder)
            MyBase.OnModelCreating(modelBuilder)
            modelBuilder.Entity(Of ModuleInfo)().ToTable("ModulesInfo")
        End Sub

        Public Property Issue() As DbSet(Of Issue)
        Public Property Task() As DbSet(Of Task)
        Public Property EFWorkflowDefinition() As DbSet(Of EFWorkflowDefinition)
        Public Property EFStartWorkflowRequest() As DbSet(Of EFStartWorkflowRequest)
        Public Property EFRunningWorkflowInstanceInfo() As DbSet(Of EFRunningWorkflowInstanceInfo)
        Public Property EFWorkflowInstanceControlCommandRequest() As DbSet(Of EFWorkflowInstanceControlCommandRequest)
        Public Property EFInstanceKey() As DbSet(Of EFInstanceKey)
        Public Property EFTrackingRecord() As DbSet(Of EFTrackingRecord)
        Public Property EFWorkflowInstance() As DbSet(Of EFWorkflowInstance)
        Public Property EFUserActivityVersion() As DbSet(Of EFUserActivityVersion)
        Public Property ModulesInfo() As DbSet(Of ModuleInfo)
    End Class
End Namespace

File: CS/WorkflowDemoEF.Module/Objects/WorkflowDemoDbContext.cs
using System;
using System.Collections.Generic;
using System.Data.Common;
using System.Data.Entity;
using System.Linq;
using System.Text;
using DevExpress.ExpressApp.EF.Updating;
using DevExpress.ExpressApp.Workflow.EF;
using DevExpress.ExpressApp.Workflow.Versioning;
using DevExpress.Workflow.EF;

namespace WorkflowDemoEF.Module.Objects {
    public class WorkflowDemoDbContext : DbContext {
        public WorkflowDemoDbContext() : base() { }
        public WorkflowDemoDbContext(string connectionString) : base(connectionString) { }
        
        protected override void OnModelCreating(DbModelBuilder modelBuilder) {
            base.OnModelCreating(modelBuilder);
            modelBuilder.Entity<ModuleInfo>().ToTable("ModulesInfo");
        }

        public DbSet<Issue> Issue { get; set; }
        public DbSet<Task> Task { get; set; }        
        public DbSet<EFWorkflowDefinition> EFWorkflowDefinition { get; set; }
        public DbSet<EFStartWorkflowRequest> EFStartWorkflowRequest { get; set; }
        public DbSet<EFRunningWorkflowInstanceInfo> EFRunningWorkflowInstanceInfo { get; set; }
        public DbSet<EFWorkflowInstanceControlCommandRequest> EFWorkflowInstanceControlCommandRequest { get; set; }
        public DbSet<EFInstanceKey> EFInstanceKey { get; set; }
        public DbSet<EFTrackingRecord> EFTrackingRecord { get; set; }
        public DbSet<EFWorkflowInstance> EFWorkflowInstance { get; set; }
        public DbSet<EFUserActivityVersion> EFUserActivityVersion { get; set; }
        public DbSet<ModuleInfo> ModulesInfo { get; set; }
    }
}
